doc/faq.rst

```diff
@@ -0,0 +1,283 @@
+   However, the code you run, such as scientific computation and deep learning models, may require additional resources beyond this baseline for Spyder itself.
+
+   .. table::
+
+      ================   ===================
+      Operating system   Version
+      ================   ===================
+      Windows            Windows 8.1
+      macOS              High Sierra (10.13)
+      Linux              Ubuntu 16.04
+      ================   ===================
+
+
+.. dropdown:: Q: How do I run Spyder installed in a conda environment using Anaconda Navigator?
+
+   Select the environment you want to launch Spyder from under :guilabel:`Applications on`.
+   If Spyder is installed in this environment, you will see it in Navigator's :guilabel:`Home` window.
+   Click :guilabel:`Launch` to start Spyder in your selected environment.
+
+   .. image:: images/faq/faq-run-environment.png
+      :alt: Navigator showing running Spyder in a specific environment
+
+
+.. dropdown:: Q: How do I run Spyder installed in a conda environment using the command line?
+
+   Activate your conda environment by typing the following in your terminal (or Anaconda Prompt on Windows):
+
+   .. code-block:: bash
+
+      conda activate <ENVIRONMENT-NAME>
+
+   Then, type ``spyder`` to launch the version installed in that environment.
+
+
+
+============
+Using Spyder
+============
+
+.. dropdown:: Q: How do I install Python packages to use within Spyder?
+
+   The first approach for installing a package should be using conda.
+   In your system terminal (or Anaconda Prompt on Windows), type:
+
+   .. code-block:: bash
+
+      conda install <PACKAGE-NAME>
+
+   If your installation is not successful, follow steps 3 through 5 of Part 2 in our `video`_ on solving and avoiding problems with pip, Conda and Conda-Forge.
+
+   .. _video: https://www.youtube.com/watch?v=Ul79ihg41Rs&t=306s
+
+
+.. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?
+
+   To work with an existing environment in Spyder, change Spyder’s default python interpreter.
+   To do so, first go to your terminal (or Anaconda Prompt on Windows), type ``conda info --envs``, and copy the path from the environment you created to your clipboard.
+
+   .. image:: images/faq/faq-conda-info.png
+      :alt: Navigator showing installing specific version of Spyder
+
+   Now, go to :guilabel:`Preferences` in Spyder’s main window, click :guilabel:`Python interpreter` and check :guilabel:`Use the following Python interpreter`.
+   Paste the path you copied before, add ``/bin/python`` at the end on Mac and Linux or ``/python.exe`` on Windows, and click :guilabel:`OK`.
+
+   .. image:: images/faq/faq-python-interpreter.png
+      :alt: Navigator showing installing specific version of Spyder
+
+   Click :guilabel:`Restart kernel` in the :guilabel:`Consoles` menu for this change to take effect.
```

Comment:
I think an screenshot is missing at this point, i.e. one that shows where the Consoles menu is placed and what's the `Restart kernel` entry you need to click on it.

Reply:
It's up to Juanita then. If she thinks this can be understood without a screenshot, I'm fine with it.

Reply:
I think as it is right now we don't need a screenshot, if we want to change it to be from the console pane then it makes sense.